Compression/src/SConscript

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+#
+# Original author Rob Kutschke.
+#
+
+import os, re
+Import('env')
+
+
+Import('mu2e_helper')
+
+helper=mu2e_helper(env)
+
+babarlibs = env['BABARLIBS']
+rootlibs  = env['ROOTLIBS']
+
+mainlib = helper.make_mainlib ( [
+    'art_Framework_Core',
+    'art_Framework_IO_ProductMix',
+    'mu2e_MCDataProducts',
+    'mu2e_RecoDataProducts',
+    'mu2e_GeneralUtilities',
+    ] )
+
+helper.make_plugins( [
+    mainlib,
+    'mu2e_Mu2eUtilities',
+    'mu2e_CaloCluster',
+    'mu2e_MCDataProducts',
+    'mu2e_RecoDataProducts',
+    'mu2e_GeometryService',
+    'mu2e_CalorimeterGeom',
+    'mu2e_CosmicRayShieldGeom',
+    'mu2e_ExtinctionMonitorFNAL_Geometry',
+    'mu2e_GlobalConstantsService_GlobalConstantsService_service',
+    'mu2e_GlobalConstantsService',
+    'mu2e_Mu2eInterfaces',
+    'mu2e_DataProducts',
+    'mu2e_GeneralUtilities',
+    'mu2e_DbService',
+    babarlibs,
+    'art_Framework_Core',
+    'art_Framework_Principal',
+    'art_Framework_Services_Registry',
+    'art_root_io_tfile_support',
+    'art_root_io_TFileService_service',
+    'art_Framework_Services_Optional_RandomNumberGenerator_service',
```

Comment:
Why does compression depend on random numbers?  Please purge unneeded libraries from the list.

Reply:
I've purged what I can in commit 3abdea0